core/services/keeper/orm_test.go

```diff
@@ -135,6 +135,139 @@ func TestKeeperDB_BatchDeleteUpkeepsForJob(t *testing.T) {
+
+	blockheight := int64(63)
+	gracePeriod := int64(10)
+
+	registry, _ := cltest.MustInsertKeeperRegistry(t, db, orm, ethKeyStore)
+
+	upkeeps := [100]keeper.UpkeepRegistration{
+		newUpkeep(registry, 0),
+		newUpkeep(registry, 1),
+		newUpkeep(registry, 2),
+		newUpkeep(registry, 3),
+		newUpkeep(registry, 4),
+		newUpkeep(registry, 5),
+		newUpkeep(registry, 6),
+		newUpkeep(registry, 7),
+		newUpkeep(registry, 8),
+		newUpkeep(registry, 9),
+		newUpkeep(registry, 10),
+		newUpkeep(registry, 11),
+		newUpkeep(registry, 12),
+		newUpkeep(registry, 13),
+		newUpkeep(registry, 14),
+		newUpkeep(registry, 15),
+		newUpkeep(registry, 16),
+		newUpkeep(registry, 17),
+		newUpkeep(registry, 18),
+		newUpkeep(registry, 19),
+		newUpkeep(registry, 20),
+		newUpkeep(registry, 21),
+		newUpkeep(registry, 22),
+		newUpkeep(registry, 23),
+		newUpkeep(registry, 24),
+		newUpkeep(registry, 25),
+		newUpkeep(registry, 26),
+		newUpkeep(registry, 27),
+		newUpkeep(registry, 28),
+		newUpkeep(registry, 29),
+		newUpkeep(registry, 30),
+		newUpkeep(registry, 31),
+		newUpkeep(registry, 32),
+		newUpkeep(registry, 33),
+		newUpkeep(registry, 34),
+		newUpkeep(registry, 35),
+		newUpkeep(registry, 36),
+		newUpkeep(registry, 37),
+		newUpkeep(registry, 38),
+		newUpkeep(registry, 39),
+		newUpkeep(registry, 40),
+		newUpkeep(registry, 41),
+		newUpkeep(registry, 42),
+		newUpkeep(registry, 43),
+		newUpkeep(registry, 44),
+		newUpkeep(registry, 45),
+		newUpkeep(registry, 46),
+		newUpkeep(registry, 47),
+		newUpkeep(registry, 48),
+		newUpkeep(registry, 49),
+		newUpkeep(registry, 50),
+		newUpkeep(registry, 51),
+		newUpkeep(registry, 52),
+		newUpkeep(registry, 53),
+		newUpkeep(registry, 54),
+		newUpkeep(registry, 55),
+		newUpkeep(registry, 56),
+		newUpkeep(registry, 57),
+		newUpkeep(registry, 58),
+		newUpkeep(registry, 59),
+		newUpkeep(registry, 60),
+		newUpkeep(registry, 61),
+		newUpkeep(registry, 62),
+		newUpkeep(registry, 63),
+		newUpkeep(registry, 64),
+		newUpkeep(registry, 65),
+		newUpkeep(registry, 66),
+		newUpkeep(registry, 67),
+		newUpkeep(registry, 68),
+		newUpkeep(registry, 69),
+		newUpkeep(registry, 70),
+		newUpkeep(registry, 71),
+		newUpkeep(registry, 72),
+		newUpkeep(registry, 73),
+		newUpkeep(registry, 74),
+		newUpkeep(registry, 75),
+		newUpkeep(registry, 76),
+		newUpkeep(registry, 77),
+		newUpkeep(registry, 78),
+		newUpkeep(registry, 79),
+		newUpkeep(registry, 80),
+		newUpkeep(registry, 81),
+		newUpkeep(registry, 82),
+		newUpkeep(registry, 83),
+		newUpkeep(registry, 84),
+		newUpkeep(registry, 85),
+		newUpkeep(registry, 86),
+		newUpkeep(registry, 87),
+		newUpkeep(registry, 88),
+		newUpkeep(registry, 89),
+		newUpkeep(registry, 90),
+		newUpkeep(registry, 91),
+		newUpkeep(registry, 92),
+		newUpkeep(registry, 93),
+		newUpkeep(registry, 94),
+		newUpkeep(registry, 95),
+		newUpkeep(registry, 96),
+		newUpkeep(registry, 97),
+		newUpkeep(registry, 98),
+		newUpkeep(registry, 99),
```

Comment:
Can we just create these in a loop instead of hard coding 100 lines?

Reply:
fixed